### PR TITLE
Remove MTI wire format

### PR DESCRIPTION
**Pull request by paulehoffman**

### CODE REVIEW

File: draft-ietf-doh-dns-over-https-latest.mkd [hunk 1]
```diff
@@ -209,15 +210,12 @@ the body parameter. The value may either be an explicit media type
...
-the request encoded with base64url {{RFC4648}}. Using the GET method is
-friendlier to many HTTP cache implementations.
-
 The DNS API Client SHOULD include an HTTP "Accept:" request header to
-say what type of content can be understood in response. The client
-MUST be prepared to process "application/dns-udpwireformat" {{dnswire}}
-responses but MAY process any other type it receives.
+say what type of content can be understood in response.
+There is no mandatory-to-implement format that clients need to understand,
+bu for maximum interoperability, the client
```
Comment (bemasc) [inline draft-ietf-doh-dns-over-https-latest.mkd]: Typo: "bu"

**Comment by mcmanus** [comment]
I'm a little undecided on the base issue here (MTI) [so leaving issue open.. perhaps f2f wg time would be good for this one] - but I've tweaked the potential PR to only deal with that. The issue where the path might not be base64url encoded was already merged separately, and the change to make the presence of a body parameter media type specific is something we don't need to do.

i've also remove the mti from the requirements section as part of the PR for consistency.

**Comment by paulehoffman** [comment]
I fully disagree with removing the MTI requirement from the doc. The purpose of the document is to transport DNS, not to be a glorified getaddrinfo() replacement.

**Comment by chantra** [comment]
isn't the lack of MIT going to be a nightmare for interoperability? Wouldn't we want all client and server to be able to speak a common format and possibly others?
Is application/dns-udpwireformat a MUST on the server side? I failed to find the requirements.

**Comment by paulehoffman** [comment]
Closing this PR and will replace it with the consensus discussion from the WG.

**Pull request closed without merging**